tests/extmod/btree1.py

```diff
@@ -56,6 +60,18 @@
 for k, v in db.items(None, None, btree.DESC):
     print((k, v))
 
+print("---")
+# R_CURSOR flag
+print(db.seq(1, b"foo1"))
+print(db.seq(1, b"qux"))
+
+try:
+    db.seq(b"foo1")
+except OSError as e:
+    assert e.args[0] == uerrno.EINVAL
```

Comment:
Will amend, thank you for the suggestions. I have see you changed the assert to print in the statvfs PR, should have done the same here.  I'll update this today.